core/src/main/java/org/apache/gravitino/storage/relational/mapper/CatalogMetaMapper.java

```diff
@@ -150,7 +150,8 @@ CatalogPO selectCatalogMetaByMetalakeIdAndName(
           + " AND metalake_id = #{oldCatalogMeta.metalakeId}"
           + " AND type = #{oldCatalogMeta.type}"
           + " AND provider = #{oldCatalogMeta.provider}"
-          + " AND catalog_comment = #{oldCatalogMeta.catalogComment}"
+          + " AND (catalog_comment = #{oldCatalogMeta.catalogComment} "
+          + "   or (catalog_comment is null and #{oldCatalogMeta.catalogComment} is null))"
```

Comment:
Can you please change to "IS NULL" for reserved words?

Reply:
DONE